api/package.json

```diff
@@ -9,16 +9,16 @@
   },
   "dependencies": {
     "@sentry/node": "5.11.1",
+    "@vercel/node": "1.7.1",
```

Comment:
How come this is moved to a regular dependency?

Reply:
Oh that was a mistake, I'll create another PR since this was just merged

Reply:
https://github.com/vercel/vercel/pull/4836